lib/algolia/client.rb

```diff
@@ -491,9 +491,17 @@ def wait_task(index_name, taskID, time_before_retry = WAIT_TASK_DEFAULT_TIME_BEF
       end
```

Comment:
You may want to add tests on insights, just to make sure that methods signatures exist.

Reply:
I'm not sure how to test that

Reply:
@julienbourdeau Just call the method, that is good just to make sure that mistakes as renaming `send_vent` are catched.